cmd/admin-group-add.go

```diff
@@ -45,9 +45,7 @@ FLAGS:
   {{end}}
 EXAMPLES:
   1. Add users 'fivecent' and 'tencent' to the group 'allcents':
-     $ set +o history
-     $ {{.HelpName}} myminio allcents fivecent tencent
-     $ set -o history
+     {{.Prompt}} {{.HelpName}} myminio allcents fivecent tencent
```

Comment:
Is there any windows equivalent to `set +o history` ?  Is it possible to print help based on the OS?  It would be good to keep set -o for linux users at least although it is not a hard requirement.  It looks like you have something that might accomplish this in cmd/admin-user-add.go?

Reply:
Its already doing that in another command in cmd/config-host-add.go - it should just use that here as well.

Reply:
I didn't do it here, because there is no sensitive information to hide here. I think disable/enable history is added here inadvertantly, hence I simply removed them leaving the only adaptive change as `{{.Prompt}}`.